src/core/utils/MultisigGraphUtils.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MultisigAccountGraphInfo } from '../../model/account/MultisigAccountGraphInfo';
+import { MultisigAccountInfo } from '../../model/account/MultisigAccountInfo';
+
+/**
+ * MultisigGraph utilities
+ */
+export class MultisigGraphUtils {
```

Comment:
None of the methods were properly documented, please consider adding docs (follow js/ts -doc pattern)

Reply:
Yes yes yes yes yes please